README.md

```diff
@@ -335,185 +360,208 @@ A range of other feedback mechanisms will be employed when appropriate including
+Please refer to the _Student Concerns, Complaints, Academic Appeals and Disciplinary Policy_ for full information on the process and procedures regarding the Disciplinary Procedures.
 
 ### Health and Safety
+
 Developers Institute will provide an environment which is safe for all students and staff.
-  - Your first responsibility is to keep yourself safe.
-  - Your second responsibility is to ensure that your actions do not endanger the safety of others.
-  - Please inform your Tutor of any hazard you identify.
-  - The First Aid resource person is your Tutor and then the Campus First Aid Officers. Your Tutor will tell you who they are.
-  - First Aid resources and procedures will be explained by your Tutor.
-  - If you have brought an approved guest on site, it is your responsibility to ensure they are aware of any hazards within the study environment
+
+- Your first responsibility is to keep yourself safe.
+- Your second responsibility is to ensure that your actions do not endanger the safety of others.
+- Please inform your Tutor of any hazard you identify.
+- The First Aid resource person is your Tutor and then the Campus First Aid Officers. Your Tutor will tell you who they are.
+- First Aid resources and procedures will be explained by your Tutor.
+- If you have brought an approved guest on site, it is your responsibility to ensure they are aware of any hazards within the study environment
 
 Your health and safety is important to us in all of the environments you encounter while training. While on your programme your health and safety is the combined responsibility of the Developers Institute, you and the owner of any premises visited as part of a training programme.
 
 Developers Institute have the responsibility to:
-  - Check for any hazards in the training venue and on extra-curricular activities or field trips, notifying you of these hazards and informing you how to minimise the risk. This includes checking the safety of any equipment used.
-  - Inform you of emergency procedures when in training venues.
-  - Ensure you have access to facilities for your health and safety at training venues.
-  - Ask you if you have any medical problems, which could increase the risk to your health and safety in any training environment.
-  - Document, investigate and report any serious accidents that occur.
-  - Understand and abide by the requirements of the *Health and Safety at Work Act 2015*.
+
+- Check for any hazards in the training venue and on extra-curricular activities or field trips, notifying you of these hazards and informing you how to minimise the risk. This includes checking the safety of any equipment used.
+- Inform you of emergency procedures when in training venues.
+- Ensure you have access to facilities for your health and safety at training venues.
```

Comment:
@Tegzma I'm not sure about your spaces, I believe there shouldn't be spaces here after each line.